<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="topVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="fromLabel">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="fromLineEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
          <property name="placeholderText">
           <string>dd/mm/yyyy</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fromTodayToolButton">
          <property name="minimumSize">
           <size>
            <width>54</width>
            <height>29</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Enter current date in &quot;from&quot; field (CTRL+F)</string>
          </property>
          <property name="text">
           <string>Today</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="toLabel">
        <property name="text">
         <string>    To:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="inputLayout">
        <item>
         <spacer name="inputSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="toLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
          <property name="placeholderText">
           <string>dd/mm/yyyy</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toTodayToolButton">
          <property name="minimumSize">
           <size>
            <width>54</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>Today</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+T</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="outputLayout">
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QLabel" name="answerLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="bottomVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../usr/share/icons/gnome/24x24/actions/exit.png</normaloff>../../../../../../usr/share/icons/gnome/24x24/actions/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFromToday">
   <property name="text">
    <string>FromToday</string>
   </property>
   <property name="toolTip">
    <string>Enter current date in &quot;from&quot; field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToToday">
   <property name="text">
    <string>To Today</string>
   </property>
   <property name="toolTip">
    <string>Enter current day in &quot;to&quot; field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionFromTextChanged">
   <property name="text">
    <string>fromTextChanged</string>
   </property>
  </action>
  <action name="actionToTextChanged">
   <property name="text">
    <string>ToTextChanged</string>
   </property>
  </action>
  <action name="actionUpdateAnswer">
   <property name="text">
    <string>updateAnswer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fromLineEdit</tabstop>
  <tabstop>fromTodayToolButton</tabstop>
  <tabstop>toLineEdit</tabstop>
  <tabstop>toTodayToolButton</tabstop>
  <tabstop>calculateButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>calculateButton</sender>
   <signal>clicked()</signal>
   <receiver>actionUpdateAnswer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromTodayToolButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFromToday</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toTodayToolButton</sender>
   <signal>clicked()</signal>
   <receiver>actionToToday</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>actionFromTextChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionToTextChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
